### DISCUSSION TITLE
Debugging/logging client side issues

### DISCUSSION THREAD

**Discussion by changsin** [cvat-ai/cvat#4149] [Q&A] [answered]
Is there a good way to debug client-side issues? We are using a local CVAT variant and the users are reporting issues that are hard to repro even with repeated tries. So we are thinking of logging on to the client-side. We can either send it via API to the server or send it to the cloud. Has anyone tried to solve similar issues? Thanks,

**Comment by nmanovic** [comment, marked as answer]
@changsin , kibana should include corresponding logs from the client. Internally we are using logs from the server and client to understand how to reproduce an issue. Also it is possible to send emails with such problems automatically (need to setup logstash correspondently).

**Reply by changsin** [reply]
Thanks for the quick response.
